pkg/db/driver_mysql.go

```diff
@@ -2,13 +2,12 @@ package db
 
 import (
 	"database/sql"
-	"fmt"
-	"net/url"
+	"net"
 	"strconv"
 
-	_ "github.com/go-sql-driver/mysql"
+	"github.com/go-sql-driver/mysql"
```

Comment:
Can you also update it to 1.8.1? It fixes a race condition with cancelled contexts: https://github.com/go-sql-driver/mysql/releases/tag/v1.8.1

Reply:
pls still update the mysql driver

Reply:
:+1: